app/views/administration/organization.html

```diff
@@ -86,7 +86,7 @@ <h4 class="list-group-item-heading"><i class="icon-calendar icon-large"></i>&nbs
                     translate}}</p>
             </a>
             <a class="list-group-item" href="#/viewpaymenttype" has-permission='READ_PAYMENTTYPE'>
-                <h4 class="list-group-item-heading"><i class="icon-pinterest icon-large"></i>&nbsp;&nbsp;
+                <h4 class="list-group-item-heading"><i class="icon-dollar icon-large"></i>&nbsp;&nbsp;
```

Comment:
MIFOSX-2323 description is about changing the icon, text for Provisioning. Why do we need to change icon for PAYMENTTYPE?